client/landing/gutenboarding/devtools.ts

```diff
@@ -27,21 +27,30 @@ export const setupWpDataDebug = () => {
 				Site.register( clientCreds );
 
 				window.wp.auth = {};
+				let previousState = window.wp?.data.select( AUTH_STORE ).getLoginFlowState();
```

Comment:
A question, because I'm not sure...

Does the optional chaining in`window.wp?` makes sense seeing as, in the [line above](https://github.com/Automattic/wp-calypso/pull/40114/files#diff-b638ce93b4149174ce9c62c84636e868R29) we're assuming that `wp` is a property of `window` anyway? (`window.wp.auth = {};`)

Reply:
Eagle eyes! No they're not needed, just a copy paste bug from when I copied code from inside the callback.

It _is_ needed inside the callback because `window.wp` could be `undefined` again by the time the callback is called (according to the type checker). Given it's just devtools it's a case of do-what-you-need-to-do-to-shut-the-type-checker-up